docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-zeebe-connection.md

```diff
@@ -0,0 +1,145 @@
+---
+id: troubleshoot-zeebe-connection
+title: "Troubleshoot Zeebe Connection Issues"
+sidebar_label: "Zeebe Connection"
+---
+
+:::note
+Web Modeler Self-Managed is available to [enterprise customers](../../../../reference/licenses.md#web-modeler) only.
+:::
+
+You try to connect (i.e., to deploy) to a remote Zeebe cluster and Web Modeler tells you
+
+- the entered cluster endpoint URL should "point to a running Zeebe cluster."
+- an unknown error occurred and to "check Zeebe cluster status."
+
+To resolve this issue, check if you can connect to Zeebe through another client, i.e.,
+[`zbctl`](../../../../apis-tools/cli-client/index.md).
+If that doesn't work, resolve the general connection issue first (see for example [the platform deployment
+troubleshooting section](https://docs.camunda.io/docs/self-managed/platform-deployment/troubleshooting/)).
+If that works, further debug your Zeebe connection with the help of the information stated below.
+Enabling [debug logging in `modeler-restapi`](#how-can-i-debug-log-grpc--zeebe-communication) may also help to
+understand the issue.
+
+## Secure connection to Zeebe fails
+
+When connecting securely to a Zeebe instance (via `https` endpoint URL), Web Modeler tries to establish a secure
+connection.
+In the process, it strictly validates the server's Application-Layer Protocol Negotiation support and its certificates
+presented against well-known certificate authorities.
+Failure to connect may have several reasons:
+
+### Solution 1: The endpoint is not configured for secure connections
```

Comment:
Isn't this rather the problem than the solution? 🙂

Reply:
You're right - I rephrased all the "solution" points to not state a problem but a solution 🙃

Reply:
Thanks :)

I'm only wondering if it could be a bit confusing for users now: In this section, the solutions address different problems, while in the section below, "Solution 1" and "Solution 2" refer to alternative solutions for the same problem.
Maybe we can just remove the "Solution" prefixes here?